models/basgra/R/run_BASGRA.R

```diff
@@ -0,0 +1,268 @@
+    "F_DIGEST_LV", "F_DIGEST_ST"      , "F_DIGEST_WALL",                   # 83:85
+    "RDRS"       , "Precipitation"    , "Nleaching"   , "NSHmob",          # 86:89
+    "NSHmobsoil" , "Nfixation"        , "Nupt"        , "Nmineralisation", # 90:93
+    "NSOURCE"    , "NSINK"            ,                                    # 94:95
+    "NRT"        , "NCRT"             ,                                    # 96:97
+    "rNLITT"     , "rNSOMF"           ,                                    # 98:99
+    "DAYL"                                                                 # 100
+  )
+  
+  outputUnits <- c(
+    "(y)"       , "(y)"      , "(d)"      , "(degC)"    , "(g C m-2)", "(g C m-2)",  #  1: 6
+    "(g DM m-2)", "(g C m-2)", "(g C m-2)", "(g C m-2)" , "(g C m-2)", "(mm)"     ,  #  7:12
+    "(m)"       , "(m2 m-2)" , "(degC)"   , "(mol m-2)" , "(-)"      , "(m)"      ,  # 13:18
+    "(m)"       , "(d)"      , "(m-2)"    , "(m-2)"     , "(mm)"     , "(mm)"     ,  # 19:24
+    "(mm)"      , "(mm)"     , "(mm)"     , "(g DM m-2)", "(g g-1)"  , "(m d-1)"  ,  # 25:30
+    "(tiller-1)", "(d-1)"    , "(m2 g-1)" , "(m-2)"     , "(-)"      , "(-)"      ,  # 31:36
+    "(-)"       , "(d-1)"    , "(-)"      ,                                          # 37:39
+    "(g C m-2)"    , "(g C m-2)"    , "(g C m-2)"    , "(g N m-2)"    , "(g N m-2)", # 40:44
+    "(g N m-2)"    , "(g N m-2)"    , "(g C m-2 d-1)", "(g N m-2 d-1)",              # 45:48
+    "(g N m-2 d-1)", "(g N m-2 d-1)", "(g N m-2 d-1)", "(-)"          ,              # 49:52
+    "(g N m-2)"    , "(g N m-2 d-1)", "(g N m-2 d-1)", "(g N m-2 d-1)", "(-)"      , # 53:57
+    "(-)"          , "(-)"          , "(-)"          ,                               # 58:60
+    "(-)", "(d-1)", "(-)", "(-)", "(d-1)", "(-)", "(m2 m-2)", "(m2 m-2)",            # 61:68
+    "(g DM m-2)"   , "(g DM m-2)"   , "(g N g-1 DM)"  ,                              # 69:71
+    "(g N m-2)"    , "(g DM m-2)"   , "(g DM m-2)"    ,                              # 72:74
+    "(g g-1 DM)"   , "(g g-1 DM)"   ,                                                # 75:76
+    "(g g-1 DM)"   , "(g g-1 DM)"   , "(g g-1 DM)"    , "(g g-1 DM)"  ,              # 77:80
+    "(-)"          , "(-)"          ,                                                # 81:82
+    "(-)"          , "(-)"          , "(-)"           ,                              # 83:85
+    "(d-1)"        , "(mm d-1)"     , "(g N m-2 d-1)" , "(g N m-2 d-1)",             # 86:89
+    "(g N m-2 d-1)", "(g N m-2 d-1)", "(g N m-2 d-1)" , "(g N m-2 d-1)",             # 90:93
+    "(g N m-2 d-1)", "(g N m-2 d-1)",                                                # 94:95
+    "(g N m-2)"    , "(g N g-1 C)"  ,                                                # 96:97
+    "(g N m-2)"    , "(g N g-1 C)"  ,                                                # 98:99
+    "(d d-1)"                                                                        # 100
+  )
```

Comment:
Not necessary for this PR, but personally, I tend to prefer code that pairs these together somehow. I'm a big fan of the` tibble::tribble` for writing inline `data.frame`s, but you could also do it with base R's `matrix(..., byrow = TRUE)`:

```r
output_vars <- matrix(c(
  "Time", "(y)",
  "year", "(y)",
  "doy", "(d)",
  ...), ncol = 2, byrow = TRUE)
```